### DISCUSSION TITLE
Home folder of ubuntu template always my_user.linux?

### DISCUSSION THREAD

**Discussion by lattice0** [lima-vm/lima#1854] [Q&A]
When I start the ubuntu template without any mounting point, the home folder is at ˋ/home/my_user.linuxˋ, but it contains nothing. Is this supposed to be the case? Shouldn't it contain Downloads, Documents, etc? Why is the home folder always my_user.linux? Is there a way to customize it?

**Comment by AkihiroSuda** [comment]
> When I start the ubuntu template without any mounting point, the home folder is at ˋ/home/my_user.linuxˋ, but it contains nothing. Is this supposed to be the case? 

Yes

> Shouldn't it contain Downloads, Documents, etc? 

No, because the image does not contain desktop environments.

> Why is the home folder always my_user.linux? 

The ".linux" suffix is appended to distinguish it from the home dir of the host (often macOS) .

> Is there a way to customize it?

https://www.google.com/search?q=change+home+directory+linux
